docs/src/__examples__/ChoiceGroup/DEFAULT.tsx

```diff
@@ -26,9 +26,21 @@ export default {
       </Stack>
     );
   },
-  info: {
-    title: "Default choice group",
-    description:
-      "Choice groups handle all changes for their children and present a single label to describe the group.",
-  },
+  exampleKnobs: [
+    {
+      component: "ChoiceGroup",
+      knobs: [
+        { name: "error", type: "text", defaultValue: "" },
+        { name: "filter", type: "boolean", defaultValue: false },
+        { name: "label", type: "text", defaultValue: "Transport to search" },
+        {
+          name: "labelAs",
+          type: "select",
+          options: ["h2", "h3", "h4", "h5", "h6"],
+          defaultValue: "h4",
+        },
```

Comment:
Same, I'd remove `labelAs`, but I'm repeating myself. I'm not sure if the purpose of the playground is to visually play with the components or not, you decide.